timemachine/cpp/src/neighborlist.cu

```diff
@@ -76,23 +76,30 @@ void Neighborlist<RealType>::compute_block_bounds_host(
 
     assert(N == N_);
     assert(D == 3);
+    int h_rebuild[1] = {1};
```

Comment:
Simpler is to just to int h_rebuild = 1; and directly take its address:

`int *d_rebuild = gpuErrchkCudaMallocAndCopy(&h_rebuild, 1*sizeof(int));`

No need to use arrays.